go-backend/db/core_value.go

```diff
@@ -0,0 +1,144 @@
+package db
+
+import (
+	"context"
+	"time"
+
+	logger "github.com/sirupsen/logrus"
+)
+
+const (
+	listCoreValuesQuery    = `SELECT * FROM core_values WHERE org_id = $1 and parent_core_value_id IS NULL`
```

Comment:
using select * is an anti pattern
give it a read here: https://stackoverflow.com/questions/321299/what-is-the-reason-not-to-use-select